app/components/guides-article.js

```diff
@@ -1,6 +1,10 @@
 import Component from '@ember/component';
 import { inject as service } from '@ember/service';
 import { computed } from '@ember/object';
+import config from '../config/environment';
+
+const addonConfig = config['guidemaker-ember-template'];
+const defaultBranch = (addonConfig && addonConfig.defaultBranch) || 'master';
```

Comment:
If the addon can support optional chaining and nullish coalescing operator, I think we can improve the code's intent by writing,

```suggestion
const defaultBranch = addonConfig?.defaultBranch ?? 'master';
```

Reply:
As this is now using the config from the guidemaker service we don't need the backing JS file for this component any more, so it was removed from the PR 👍